diff --git a/bundle/vim-markdown-toc/ftplugin/markdown.vim b/bundle/vim-markdown-toc/ftplugin/markdown.vim
index c8bcaaf..8b57d71 100644
--- a/bundle/vim-markdown-toc/ftplugin/markdown.vim
+++ b/bundle/vim-markdown-toc/ftplugin/markdown.vim
@@ -424,6 +424,7 @@ function! s:UpdateToc()
     endif
 
     call winrestview(l:winview)
+    syn sync fromstart
 endfunction
 
 function! s:DeleteExistingToc()
