diff --git a/.vim/bundle/vim-ollama/plugin/ollama.vim b/.vim/bundle/vim-ollama/plugin/ollama.vim
index 98554cd..9aa242c 100644
--- a/.vim/bundle/vim-ollama/plugin/ollama.vim
+++ b/.vim/bundle/vim-ollama/plugin/ollama.vim
@@ -123,17 +123,18 @@ function! s:MapTab() abort
     vnoremap <Plug>(ollama-edit)           <Cmd>call ollama#edit#EditPrompt()<CR>
 
     " Setup default mappings
-    imap <silent> <C-]>     <Plug>(ollama-dismiss)
-    imap <silent> <Tab>     <Plug>(ollama-tab-completion)
-    imap <silent> <M-Right> <Plug>(ollama-insert-line)
-    imap <silent> <M-C-Right> <Plug>(ollama-insert-word)
-    vmap <silent> <leader>r <Plug>(ollama-review)
-    nmap <silent> <C-M-y> <Plug>(ollama-accept-changes)
-    nmap <silent> <C-M-n> <Plug>(ollama-reject-changes)
-    nmap <silent> <C-Y> <Plug>(ollama-accept-all-changes)
-    nmap <silent> <C-N> <Plug>(ollama-reject-all-changes)
-    nmap <silent> <C-I> <Plug>(ollama-edit)
-    vmap <silent> <C-I> <Plug>(ollama-edit)
+    "
+"    imap <silent> <C-]>     <Plug>(ollama-dismiss)
+"    imap <silent> <Tab>     <Plug>(ollama-tab-completion)
+"    imap <silent> <M-Right> <Plug>(ollama-insert-line)
+"    imap <silent> <M-C-Right> <Plug>(ollama-insert-word)
+"    vmap <silent> <leader>r <Plug>(ollama-review)
+"    nmap <silent> <C-M-y> <Plug>(ollama-accept-changes)
+"    nmap <silent> <C-M-n> <Plug>(ollama-reject-changes)
+"    nmap <silent> <C-Y> <Plug>(ollama-accept-all-changes)
+"    nmap <silent> <C-N> <Plug>(ollama-reject-all-changes)
+"    nmap <silent> <C-I> <Plug>(ollama-edit)
+"    vmap <silent> <C-I> <Plug>(ollama-edit)
 endfunction
 
 " Create autocommand group
diff --git a/.vim/bundle/vim-ollama/python/OllamaLogger.py b/.vim/bundle/vim-ollama/python/OllamaLogger.py
index 0b6b3ed..86bb7af 100644
--- a/.vim/bundle/vim-ollama/python/OllamaLogger.py
+++ b/.vim/bundle/vim-ollama/python/OllamaLogger.py
@@ -17,7 +17,7 @@ class OllamaLogger:
 
     def __init__(self, log_filename, log_level=logging.ERROR):
         # Create a logger
-        self.logger = logging.getLogger()
+        self.logger = logging.getLogger(__name__)
         self.logger.setLevel(log_level)
 
         # Attempt to create a log directory and set up file logging
